### PR TITLE
Display an alert instead of an empty table

### LINKED ISSUE #155: Display an alert instead of an empty table
* Pages displaying any data should display an alert instead of an empty table when there is no data to be displayed.

### PR DESCRIPTION
**Pull request by xkostka2**
* Added alert message, if table is empty, to tables where it was missing.
* Added alert message to list components, where after applying filter, no data are shown.
* Resolves #155 .

### CODE REVIEW

**Comment by Vojtech-Sassmann** [comment]
Hi @xkureck could you please provide a review for this PR?

File: src/app/vos/pages/vo-detail-page/vo-groups/vo-groups.component.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h1 class="page-subtitle">{{'VO_DETAIL.GROUPS.TITLE' | translate}}</h1>
     {{'VO_DETAIL.GROUPS.LIST_VIEW' | translate}}
   </mat-checkbox>
   <mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>
-  <div class="mt-3" *ngIf="!loading">
+  <div class="mt-3" *ngIf="!loading && groups.length !== 0">
```
Comment (xkureck) [inline src/app/vos/pages/vo-detail-page/vo-groups/vo-groups.component.html]: I suggest to take condition groups.length !== 0 and alert of no groups into components app-groups-tree & app-groups-list, to reduce duplicity of code, if you do that you don't have to add this alert everywhere you use these components

File: src/app/admin/components/attr-def-list/attr-def-list.component.html [hunk 2]
```diff
@@ -55,3 +55,11 @@
     <mat-paginator [length]="this.definitions.length" [pageSize]="10" [pageSizeOptions]="[5,10,25]"></mat-paginator>
   </div>
 </div>
+
+<div *ngIf="definitions.length === 0" class="perun-alert warn-alert">
+  {{'VO_DETAIL.GROUPS.NO_GROUPS' | translate}}
```
Comment (Vojtech-Sassmann) [inline src/app/admin/components/attr-def-list/attr-def-list.component.html]: This does not seem to be a valid value for this text.

**Comment by Vojtech-Sassmann** [comment]
Hi @xkostka2. Could you use everywhere the `[hidden]` property instead of `ngIf` (in the same use-case, not everywhere in the project)? We could prevent some future bugs.

**Pull request merged**